lti_consumer/utils.py

```diff
@@ -31,6 +31,14 @@ def _(text):
     return text
 
 
+def ngettext_fallback(text_singular, text_plural, number):
```

Comment:
@ahmadabuwardeh Looking at this change it makes sense to have in a separate pull request.

However, I'd like to understand the motivation behind it and if it's crucial for this pull request.